pages/workspaces/[workspaceId]/index.tsx

```diff
@@ -90,62 +91,65 @@ const WorkspaceDashboard = ({ workspace }: WorkspaceDashboardProps) => {
   }, [repositories, filteredRepositories]);
 
   return (
-    <WorkspaceLayout workspaceId={workspace.id}>
-      <WorkspaceHeader workspace={workspace} />
-      <div className="grid sm:flex gap-4 pt-3">
-        <WorkspacesTabList workspaceId={workspace.id} selectedTab={"repositories"} />
-        <div className="flex justify-end items-center gap-4">
-          <Button variant="outline" onClick={() => router.push(`/workspaces/${workspace.id}/settings#load-wizard`)}>
-            Add repositories
-          </Button>
-          <DayRangePicker />
-          <TrackedRepositoryFilter
-            options={filterOptions}
-            handleSelect={(selected: OptionKeys[]) => setFilteredRepositories(selected)}
-          />
+    <>
+      <WorkspaceOgImage workspace={workspace} range={range} />
```

Comment:
This is the main change. The rest is just realignment of spacing because things are in a fragment now.